scripts/zones/AlTaieu/npcs/qm2.lua

```diff
@@ -1,25 +1,26 @@
 -----------------------------------
 -- Area: Al'Taieu
--- NPC:  ??? (Jailer of Justice Spawn)
+--  NPC: ??? (Jailer of Justice Spawn)
 -- Allows players to spawn the Jailer of Justice by trading the Second Virtue, Deed of Moderation, and HQ Xzomit Organ to a ???.
 -- @pos , -278 0 -463
 -----------------------------------
 package.loaded["scripts/zones/AlTaieu/TextIDs"] = nil;
 -----------------------------------
-
 require("scripts/zones/AlTaieu/TextIDs");
 
 -----------------------------------
 -- onTrade Action
 -----------------------------------
 
-function onTrade(player,npc,trade)  
+function onTrade(player,npc,trade)
+    --[[
     -- Trade the Second Virtue, Deed of Moderation, and HQ Xzomit Organ
-    --[[if (GetMobAction(16912839) == 0 and trade:hasItemQty(1853,1) and trade:hasItemQty(1854,1) and trade:hasItemQty(1785,1) and
+    if (GetMobAction(16912839) == 0 and trade:hasItemQty(1853,1) and trade:hasItemQty(1854,1) and trade:hasItemQty(1785,1) and
     trade:getItemCount() == 3) then
         player:tradeComplete();
-        SpawnMob(16912839,900):updateClaim(player); -- Spawn Jailer of Justice
-    end]]
+        SpawnMob(16912839):updateClaim(player); -- Spawn Jailer of Justice
+    end
+    ]]
```

Comment:
Yeah I just moved the placement of the comment block to be what I thought was more legible/less likely to have its indents f\*cked later